### DISCUSSION TITLE
GraphRAG Output Uses

### DISCUSSION THREAD

**Discussion by davidgross631** [microsoft/graphrag#933] [Q&A] [answered]
Hi, I have been experimenting with GraphRAG and have been using it to organize big chunks of data into a knowledge graph and ask questions about it (quintessentially what GraphRAG should be used for), however I have reached a problem/question. 

How would one return/output that knowledge graph? Or morever, a json or text based representation of all of the data and how it's organized? It seems like GraphRAG can't really output alot of data. I have experimented with different prompts but still have the same issue. 

So, another question also would be, is GraphRAG really the tool to use for this? Is it meant only for questions that have shorter outputs or?

**Comment by natoverse** [comment, marked as answer]
This might have some helpful comments: https://github.com/microsoft/graphrag/discussions/697

Essentially, we can output the underlying graph structure in such a way that you can plot it or build UIs to navigate it.
